src/json/members/kygchng.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Kylie Chang",
+    "pennkey": "kygchng",
+    "major": "CIS + Finance",
+    "school": "Wharton + SEAS",
+    "bio": "Student at M&T/lover of matcha, sweetgreen, and applesauce.",
+    "hometown": "San Jose, CA",
+    "team": "OHQ",
+    "roles": ["Frontend Engineer"],
+    "photo": "https://imgur.com/ThkDgjq",
```

Comment:
This is a website url, not an image URL. You need to right click on the photo and select "Copy Image Address".